validator/src/admin_rpc_service.rs

```diff
@@ -303,6 +350,40 @@ impl AdminRpc for AdminRpcImpl {
         AdminRpcImpl::set_identity_keypair(meta, identity_keypair, require_tower)
     }
 
+    fn set_relayer_config(
+        &self,
+        meta: Self::Metadata,
+        relayer_url: String,
+        trust_packets: bool,
+        expected_heartbeat_interval_ms: u64,
+        max_failed_heartbeats: u64,
+    ) -> Result<()> {
+        debug!("set_relayer_config request received");
```

Comment:
```suggestion
        debug!("set_relayer_config() request received");
```

Reply:
gonna leave as is to match the existing style